ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -600,7 +600,7 @@ private object OracleQueries extends Queries {
     bigIntType ++ sql" GENERATED ALWAYS AS IDENTITY"
   protected[this] override def textType = sql"NVARCHAR2(100)"
   protected[this] override def packageIdType = sql"NVARCHAR2(64)"
-  protected[this] override def partyOffsetContractIdType = sql"NVARCHAR2(255)"
+  protected[this] override def partyOffsetContractIdType = sql"VARCHAR2(255)"
```

Comment:
Search indices may not refer to nvarchar2 columns; this shouldn't make a difference for party IDs, offsets, or contract IDs.